_includes/specials/bech32/22-priority.md

```diff
@@ -0,0 +1,35 @@
+We've frequently mentioned the fee savings available to people spending
+segwit inputs, but we've never before mentioned that you don't need to
+take advantage of the savings.  If you want, you can pay the same fee
+you would've paid without segwit in order to possibly have your
+transaction confirm quicker (all other things being equal).  For some
```

Comment:
nit: iirc "quicker" is for comparison and would be followed by "than"; perhaps "more quickly" instead?